### PR TITLE
Consolidate all scripts into build.gradle

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

File: scripts/google-java-format.gradle [hunk 1]
```diff
@@ -1,6 +0,0 @@
-apply plugin: 'com.github.sherter.google-java-format'
-
-check.dependsOn verifyGoogleJavaFormat
-
-// to auto-format run ./gradlew googleJavaFormat
```
Comment (coollog) [inline scripts/google-java-format.gradle:5]: Should this comment be added above too?

File: scripts/publishing.gradle [hunk 2]
```diff
@@ -1,60 +0,0 @@
-apply plugin: 'com.bmuschko.nexus'
```
Comment (coollog) [inline scripts/publishing.gradle:1]: This script is just removed?

**Review by coollog — APPROVED** [review]

File: scripts/publishing.gradle (hunk 2 above)
Comment (loosebazooka) [reply scripts/publishing.gradle:1]: Yeah that was the old maven release plugin, but now we go through the new flow.

File: scripts/google-java-format.gradle (hunk 1 above)
Comment (loosebazooka) [reply scripts/google-java-format.gradle:5]: sure, lemme add it back in.

**Review by coollog — APPROVED** [review]

**Pull request merged**